modules/tests/test/modify-vm-template_test.go

```diff
@@ -3,26 +3,70 @@ package test
 import (
 	"context"
 
-	"github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/zerrors"
+	"github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/zutils"
 	testtemplate "github.com/kubevirt/kubevirt-tekton-tasks/modules/sharedtest/testobjects/template"
 	. "github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/constants"
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/framework"
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/runner"
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/testconfigs"
 	. "github.com/onsi/ginkgo"
-	templatev1 "github.com/openshift/api/template/v1"
 	kubevirtv1 "kubevirt.io/client-go/api/v1"
 
 	"github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
+	templatev1 "github.com/openshift/api/template/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 )
 
+type ExpectedResults struct {
+	VM       *kubevirtv1.VirtualMachine
```

Comment:
semantically having VM and Template doesn't seem right to have them here. Please either pass them as a parameter to the expect functions or rename the object.